### ISSUE TITLE
OG html tags for links on Twitter/Facebook etc

### ISSUE THREAD

**Issue by eddiejaoude** [DashboardHub/PipelineDashboard#627]
### Requirements

When sharing DashboardHub links on social media, rather than the media platform selecting the text and picture, we can specify this in the meta tags.

**UI**

- html meta tags
- on project dashboard page, use project icon, if no icon available, use dashboardhub logo

**Comment by webkhushboo** [comment]
@eddiejaoude 

I found ngx-meta plugin to update the project icon while sharing link on facebook/twitter.
Using this meteservice we can update the tags dynamically

Please have a look on the below link and let me know your inputs on it
https://www.npmjs.com/package/@ngx-meta/core

**Comment by eddiejaoude** [comment]
I wonder if we can use something like this? https://angular.io/guide/set-document-title

**Comment by webkhushboo** [comment]
There is service for adding and removing metatags too. 
https://angular.io/api/platform-browser/Meta

 https://angular.io/guide/set-document-title This service seems to be for updating title of the page

**Comment by eddiejaoude** [comment]
Ok great, so the 3rd party library is no longer needed, we can use the core angular services 👍

**Comment by webkhushboo** [comment]
@eddiejaoude 
Please provide the image size 200*200 for square size image in the facebook/twitter share.
Format for image should be in png. It doesn't support svg images.

Please find the below dimensions.
The minimum allowed image dimension is 200 x 200 pixels.
The size of the image file must not exceed 8 MB.
Use images that are at least 1200 x 630 pixels for the best display on high resolution devices. At the minimum, you should use images that are 600 x 315 pixels to display link page posts with larger images.
https://developers.facebook.com/docs/sharing/webmasters/images/

**Comment by webkhushboo** [comment]
@eddiejaoude 

I am able to show the thumbnail in square shape on twitter using twitter card `summary` while on facebook I have used `og:type="article"`. Its still showing the full image as image size is big.

So I found this below for showing the square thumbnail on facebook and linkedin:
`Images that are at least 470×246 pixels will display as a rectangle with the title and description below. Images that are smaller than 470×246 pixels will display as a square with the title and description to the right.`
